Angular - Frontend/src/app/Services/download-file.service.ts

```diff
@@ -0,0 +1,25 @@
+import {Injectable} from '@angular/core';
+import {HttpClient, HttpEvent, HttpRequest} from '@angular/common/http';
```

Comment:
remove  HttpEvent, HttpRequest if you don't use

Reply:
done